options.ts

```diff
@@ -50,7 +50,6 @@ export class OptionsBase {
 		return {
 			all: { type: OptionType.Boolean },
 			json: { type: OptionType.Boolean },
-			watch: { type: OptionType.Boolean },
```

Comment:
Note that I changed its default value to **true** because this should be its default value in CLI now. Is this OK with appbuilder?